### PR TITLE
Move to SQS

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: server/services/auditService.test.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  return {
+    SQSClient: jest.fn(() => sqsClientInstance),
+    SendMessageCommand,
+  }
+})
+
+describe('Audit service', () => {
+  let auditService: AuditService
+
+  describe('getVisitors', () => {
```
Comment (tpmcgowan) [inline server/services/auditService.test.ts:18]: is `getVisitors` the right label for this?

File: server/routes/search.ts [hunk 2]
```diff
@@ -53,6 +55,8 @@ export default function routes(
     const realNumberOfResults = errors.length > 0 ? 0 : numberOfResults
     const currentPageMax = parsedPage * pageSize
     const to = realNumberOfResults < currentPageMax ? realNumberOfResults : currentPageMax
+    auditService.prisonerSearch(search, 'HEI', res.locals.user?.username)
```
Comment (tpmcgowan) [inline server/routes/search.ts:58]: Is it worth adding to the test for this route a check that audit called?

**Review by tpmcgowan — DISMISSED** [review]
LGTM - like how this is done! Just two minor comments.

File: server/services/auditService.test.ts (hunk 1 above)
Comment (psoleckimoj) [reply server/services/auditService.test.ts:18]: Whoops 😆

File: server/routes/search.ts (hunk 2 above)
Comment (psoleckimoj) [reply server/routes/search.ts:58]: Will have a nosey!

**Pull request merged**